com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -710,7 +710,7 @@ void _AgentReset()
             AgentReset();
         }
 
-        public void UpdateAgentAction(AgentAction action)
+        internal void UpdateAgentAction(AgentAction action)
```

Comment:
Should we just remove this and UpdateVectorAction? I don't think they're used anywhere currently.

Reply:
Removed it